SDBLV/beliveo-files
# modify menu to apps

#### What does this PR do?
- add components in a menu bar 

#### Where should the reviewer start?
- beliveo-files -> components -> menuBar.php
- beliveo-files -> styles -> main.css

#### How should this be manually tested? (if applies)
1. :one: verify if the code on app-template works once we add a div to add the component
